test/FunctionalTests/Microsoft.OData.Edm.Tests/Csdl/CsdlReaderTests.cs

```diff
@@ -1230,5 +1231,323 @@ public void ValidateEdmxVersions(string odataVersion)
             Assert.Empty(edmErrors);
             Assert.Equal(edmModel.GetEdmVersion(), odataVersion == "4.0" ? EdmConstants.EdmVersion4 : EdmConstants.EdmVersion401);
         }
+
+        [Fact]

```

Comment:
>Fact [](start = 9, length = 4)

Do you have any tests that validate referenced types (not annotations) from referenced namespace.

For example, in your tests, you could test loading permissionsCsdl and validating that it imported types from "Default" and "Exmaple.Types" from main.csdl, but didn't import types from main.csdl defined in 3rd namespace.

Reply:
I've added the test.